privatek8s.tf

```diff
@@ -91,7 +91,8 @@ resource "azurerm_kubernetes_cluster_node_pool" "infracipool" {
     "kubernetes.azure.com/scalesetpriority" = "spot"
   }
   node_taints = [
-    "kubernetes.azure.com/scalesetpriority=spot:NoSchedule"
+    "jenkins=infra.ci.jenkins.io:NoSchedule",
```

Comment:
Why adding this taint? Isn't the kubernetes one enough?

Reply:
Once we'll move release.ci.jenkins.io in this cluster, we'll have to ensure that their pods are not scheduled on the same nodes as infra.ci.
Hence the toleration.